File: common/src/main/java/de/werkelmann/rekote/address/ip/IpChecker.java
package de.werkelmann.rekote.address.ip;

import de.werkelmann.rekote.address.NotEmptyRule;
import de.werkelmann.rekote.address.ServerConnectionChecker;

public class IpChecker extends ServerConnectionChecker {

    public IpChecker() {
        rules.add(new NotEmptyRule());
        rules.add(new IsIpRule());
    }
}

File: common/src/main/kotlin/de/werkelmann/rekote/address/ip/IpChecker.kt
package de.werkelmann.rekote.address.ip

import de.werkelmann.rekote.address.NotEmptyRule
import de.werkelmann.rekote.address.ServerConnectionChecker

class IpChecker : ServerConnectionChecker() {
    init {
        rules.add(NotEmptyRule())
        rules.add(IsIpRule())
    }
}
